scripts/get_untriaged.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import csv
+import json
+import os
+import sys
+from datetime import datetime
+from datetime import date
+from datetime import timedelta
+
+
+# Inject project path
+sys.path.append('../')
+
+
+def fetch_untriaged(args):
+    from bugbug import bugzilla
+    today = date.today() - timedelta(days=2)
+    three_months_ago = today - timedelta(days=args.days_back)
+
+    # Set bugzilla token and download bugs
+    bugzilla.set_token(args.token)
+    bugzilla.download_bugs_between(three_months_ago, today)
+
+    # Get untriaged bugs
+    bugs = bugzilla.get_bugs()
+    untriaged_bugs = []
+    for bug in bugs:
```

Comment:
Can you make sure we only return the bugs we've just downloaded? (bugzilla.get_bugs() will get all bugs from the bugs DB).

You can do that by simply storing the bug IDs when you call download_bugs_between, and add a if bug['id'] not in bug_ids: continue here